web_development_101/the_basics/join_the_odin_community.md

```diff
@@ -24,11 +13,15 @@ Remember that project you struggled so hard to figure out and that you're so pro
 
 We are working hard to update existing lessons and produce new content, so we would love to hear your feedback on the lessons and projects. We hope you find the lessons fun, engaging, and informative and find the projects challenging but achievable. So please let us know your thoughts!
 
-### Asking for help
+### Before Asking for help
 
 As most of the projects are designed to push you to your limit, please remember that there is always a community to turn to! You don't need to know how to solve every problem straight away, BUT you do need to have a general idea of where you are going. This becomes really important when asking your questions because sometimes the problem is your **approach** and not your code.
 
-If you're feeling stuck, it's a good time to pause and take a breather. Break the problem down into little pieces and then decide what is really holding you back. We call this technique [rubber duck debugging](https://en.wikipedia.org/wiki/Rubber_duck_debugging). If this method doesn't yield a solution for you, it's time to reach out to the community and ask for help.
+If you're feeling stuck, it's a good time to pause and take a breather. Break the problem down into little pieces and then decide what is really holding you back. We call this technique [rubber duck debugging](https://en.wikipedia.org/wiki/Rubber_duck_debugging). You can also try doing a [google search](https://www.google.com/). Google is a very powerful tool that can present relevant information, depending on what you enter. Read this post about [How to use Google to solve your programming questions](https://codinginflow.com/google-programming-questions). You could also look back at relevant lessons for tools to take on the current task. If these methods don't yield a solution for you, it's time to reach out to the community and ask for help.
+
+### Asking for help
+
+One of the first things to keep in mind is that [You don't need to ask to ask](https://dontasktoask.com/). While a simple idea, it can help you get answers to your questions a lot faster. However, you'll want to make sure that you help the community help you by putting together a great question.
```

Comment:
```suggestion
So you've spent some time struggling to solve the problem on your own, and now it's time to fire up the Odin Discord and ask for help. The first thing to keep in mind is [don't ask to ask](https://dontasktoask.com/). While this a simple idea (with a pretty catchy motto!), it can help you get answers to your questions much faster and will make it easier to others to feel comfortable offering you help.

In addition, when you ask your questions, you should help the community help you by putting together a great question.
```